vrc-get-gui/app/settings/page.tsx

```diff
@@ -225,14 +225,17 @@ function Settings(
 			</Card>
 			<Card className={"flex-shrink-0 p-4"}>
 				<h2>{tc("backup")}</h2>
-				<h3>{tc("backup path")}</h3>
-				<Typography className={"whitespace-normal"}>
-					{tc("the backup path is the directory where vrc-get-gui will create backup zips of the projects.")}
-				</Typography>
-				<div className={"flex gap-1"}>
-					<Input className="flex-auto" value={settings.project_backup_path} disabled/>
-					<Button className={"flex-none px-4"} onClick={selectProjectBackupFolder}>{tc("select")}</Button>
+				<div className="mt-2">
+					<h3>{tc("backup path")}</h3>
+					<Typography className={"whitespace-normal"}>
+						{tc("the backup path is the directory where vrc-get-gui will create backup zips of the projects.")}
+					</Typography>
+					<div className={"flex gap-1"}>
+						<Input className="flex-auto" value={settings.project_backup_path} disabled/>
+						<Button className={"flex-none px-4"} onClick={selectProjectBackupFolder}>{tc("select")}</Button>
+					</div>
 				</div>
+				<div className="mt-2">
```

Comment:
please add indent

Reply:
I'm not sure i understand what you mean.

This class adds two units of margin top to the component so it can be further from the section title.

What would you want to do here ?

Reply:
Did you mean to add some padding left so the text can be pushed to the right ?

Reply:
please add indent to line 239-247

Reply:
I don't know how i missed that and didn't understand :/ sorry